config.js

```diff
@@ -423,6 +423,10 @@ var config = {
     // user documentation.
     // userDocumentationURL: 'https://docs.example.com/video-meetings.html'
 
+    // If specified a 'Download our apps' button will be displayed in the overflow menu with a link
+    // to the specified URL for an app download page.
+    // downloadAppsUrl: 'https://docs.example.com/our-apps.html'
```

Comment:
Let's create a section for these links instead of having them at the top-level. `deploymentUrls` maybe? @hristoterezov WDYT?

Reply:
OK